src/theme/index.tsx

```diff
@@ -20,6 +20,14 @@ export const MEDIA_WIDTHS = {
   upToLarge: 1280,
 }
 
+export enum Z_INDEX {
+  background = -1,
+  zero = 0,
```

Comment:
Can this be given a more semantic value?

Reply:
yeah i was hoping to have a bit of a CSS style conversation on how to approach this. i'm definitely in favor of putting some standard convention around z-index (https://getbootstrap.com/docs/5.0/layout/z-index/, https://tailwindcss.com/docs/z-index), but i think that is outside of the scope of this bug fix

i would hope that convention leads to never using zero - but it seemed like a reasonable choice to get this bug fixed and deployed

Reply:
I would be in favor of bootstrap's approach. I'd probably make the change in this PR too so people don't start relying on temporary `Z_INDEX` values.